File: Örnek-6.6/Matlab/dortgenalan.m
e=input('Birinci köşegeni giriniz (cm): ');
f=input('İkinci köşeggeni giriniz (cm): ');
aci=input('Arasındaki açıyı giriniz (derece): ');

alan=e*f*sind(aci)/2;

fprintf('Dörtgenin alanı: %0.3f',alan);

File: Örnek-6.6/C/dortgenalan.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>



int main() {
	float e,f,aci,alan;
	printf("Birinci kosegenin uzunlugunu giriniz (cm): ");
	scanf("%f",&e);
	printf("ikinci kosegenin uzunlugunu giriniz (cm): ");
	scanf("%f",&f);
	printf("Aradaki aciyi giriniz (derece): ");
	scanf("%f",&aci);
	
	alan=e*f*sin(M_PI*aci/180)/2;
	
	printf("Dortegenin alani: %0.3f",alan);
	
	return 0;
}
